pkg/workflows/wasm/host/standard_test.go

```diff
@@ -138,10 +143,37 @@ func TestStandardModeSwitch(t *testing.T) {
 
 		m := makeTestModule(t)
 		request := triggerExecuteRequest(t, 0, &basictrigger.Outputs{CoolOutput: anyTestTriggerValue})
-
 		result := executeWithResult[string](t, m, request, mockExecutionHelper)
-
 		require.Equal(t, "test556", result)
+
+		// Verify time call sequence is Node --> DON --> Node --> DON
```

Comment:
This may not always be true, it's language dependant. Go does it during initialization. Before the first DON mode, we can just ignore it.

Reply:
I wonder if the easiest thing to do is to have three flags below initialized to false.

donCall1
nodeCall
donCall2

In the don time call, if nodeCall is true, set donCall2 true. Otherwise, set donCall1 true

In node call, if donCall1 is true, set nodeCall to true. Here, you can leave a comment about initialization. 

At the end, check all three are true.